src/openzaak/utils/permissions.py

```diff
@@ -188,3 +189,9 @@ def has_object_permission(self, request: Request, view, obj) -> bool:
         main_object_data = self.format_data(main_object, request)
         fields = self.get_fields(main_object_data)
         return request.jwt_auth.has_auth(scopes_required, component, **fields)
+
+
+class ImportAuthRequired(AuthRequired):
+    def get_component(self, view) -> str:
+        importer_type = view.import_type
+        return ImportTypeChoices.get_component_from_choice(importer_type)
```

Comment:
It looks like overkill now, before Import process becomes generic. 

Let's also discuss permissions. What happens if you try to import a document which has an informatieobjecttype which you don't have rights for? 
Should we stop the whole process or should we just not import this particular document? I think it should be refined with Joeri